src/bindings/python/src/pyopenvino/CMakeLists.txt

```diff
@@ -57,7 +57,8 @@ else()
 endif()
 
 target_include_directories(${PROJECT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/..")
-target_link_libraries(${PROJECT_NAME} PRIVATE openvino::runtime ${OFFLINE_TRANSFORMATIONS_LIB} openvino::frontend::common)
+target_link_libraries(${PROJECT_NAME} PRIVATE openvino::runtime ${OFFLINE_TRANSFORMATIONS_LIB} openvino::frontend::common
+        nlohmann_json)
```

Comment:
`nlohmann_json` should be public dependency of `offline_transformations`, then you don't need it here explicitly

Reply:
fixed